deployment/articles/04-securing-liferay/05-ldap/01-ldap-servers-intro.markdown

```diff
@@ -135,12 +135,12 @@ method you use. For example, if you changed @product@'s authentication method to
 use screen names instead of the email addresses, you would modify the search
 filter so it can match the entered log in name:
 
-        (cn=@screen_name@)
+    (cn=@screen_name@)
 
```

Comment:
@mwilliams2014 Something's still not right with this.These LDAP queries are code; they should be formatted as such (these are docs I wrote).

Reply:
@sez11a it's still formatted as code. I believe this one was just indented one tab too many

Reply:
I removed the extra space in the code-block

Reply:
@sez11a see the extra space here: https://portal.liferay.dev/docs/7-2/deploy/-/knowledge_base/d/ldap#users